test/Scripts.Integration.Test/symbol-upload-server.py

```diff
@@ -12,6 +13,20 @@
 apiProject = 'sentry-unity'
 uri = urlparse(sys.argv[1] if len(sys.argv) > 1 else 'http://127.0.0.1:8000')
 uploads = {}
+lock = Lock()
+
+
+def registerUpload(name: str, chunks: int):
```

Comment:
Does it make sense to change our "expected" count of debug symbols with a boolean to check if anything got uploaded at all? To get rid of those random changes?

Reply:
I wouldn't. I know it's not very convenient to update right now, but at least you know it's doesn't break completely (like uploading just some small bit, single chunk instead of 6, for example) by some change we do or a dependency update.